### ISSUE TITLE
xpos tag typos

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Italian-VIT#1]
I found a bunch of xpos tag typos in this dataset as well.  I contacted Prof. Simi, but I don't know whether this is an issue in the conversion or the underlying data, so I will file a bug here as well.

```
# sent_id = VIT-2110
# text = Tra queste figura appunto il 60% della Cesi e quindi, indirettamente, il controllo della Merone.

13      indirettamente  indirettamente  ADV     16      _       3       advmod  _       SpaceAfter=No

xpos tag is 16, should be B
-----------------------------
# sent_id = VIT-8538
# text = Strette di mano, autografi, evviva all'indirizzo del sindaco e naturalmente di "Le Pen presidente".

15      naturalmente    naturalmente    ADV     18       _       18      advmod  _       _

xpos tag is 18, should be B
-----------------------------
# sent_id = VIT-3949
# text = Naturalmente, il sistema si allungava in orizzontale.

8       orizzontale     orizzontale     ADV     6       Number=Sing     6       advmod  _       SpaceAfter=No

xpos tag is 6, should be B
-------------------
# sent_id = VIT-787
# text = Per il gioco delle candidature assegnate a tavolino nei vari collegi all'una o all'altra forza (grosso modo due terzi in quota alla Lega e un terzo in quota a FI), il risultato maggioritario non si è poi distribuito sulla base dei rispettivi pesi ipotizzati in apertura di campagna elettorale ed è venuto a favorire i candidati Lega e An, e per contro a danneggiare i candidati FI.

23      grosso  grosso  ADJ     a       Gender=Masc|Number=Sing 24      amod    _       _

xpos tag is "a", should be "A"
------------------------
# sent_id = VIT-5937
# text = Nel corso di un'esperienza durata più di due anni, sono stati i fatti a mettere in evidenza - in particolare con il decreto sul costo del lavoro, coll'installazione dei missili a Comiso, coi tagli alle spese sociali, coll'avvallo alle scelte di pentapartito per le giunte - che questo governo approfondiva le divisioni a sinistra e nello schieramento riformatore e favoriva, invece come infatti è accaduto, la ripresa democristiana, e che la lotta anche aspra contro una politica giudicata erronea non ha mai impedito ai comunisti il sostegno di misure o di gesti positivi.

9       di      di      CCONJ   ADP     _       8       fixed   _       _

xpos tag is ADP, should be CC
--------------------------
# sent_id = VIT-4973
# text = Come si fa a giudicare il ruolo svolto dal Cnuce di Pisa (che non produce tante "pubblicazioni", e cioè "servizi" sofisticati) per mezzo di aride tabelle?

24      cioè    cioè    CCONJ   ADV     _       26      advmod  _       _

xpos tag is ADV.  This one confuses me - could be CCONJ/CC or ADV/B, but I don't know
--------------------------
# sent_id = VIT-5936
# text = Al momento dell'incarico a Craxi nel 1983 il Pci ha sottolineato, anzi, che tale incarico era il risultato della sconfitta inferta al tentativo di spostamento a destra compiuto dalla Dc e ha perciò dichiarato di attendere il governo alla prova dei fatti.

17      anzi    anzi    CCONJ   ADV     _       15      advmod  _       SpaceAfter=No

xpos tag is ADV.  This one I also do not know how to resolve.
--------------------------
# sent_id = VIT-7435
# text = Scusate, dicevano, ma intanto continuavano con il tam tam: "speriamo che i cinghiali lo sentano, e se ne stiano rintanati", ridacchiavano fra di loro.

6       intanto intanto CCONJ   ADV     _       7       advmod  _       _

Again, I do not know which would be better, CCONJ/CC or ADV/B
---------------------------
# sent_id = VIT-922
# text = Non abbiamo fatto altro, in questi mesi, che offrire questi spazi al mercato e saturare così la nostra offerta.

10      che     che     SCONJ   Cs      _       11      mark    _       _

Woo, back to easy ones.  This one should be CS
------------------------
# sent_id = VIT-7045
# text = Il sidecar di Totò ne "tartassati" e a completare questa piccola antologia di cinema su due e tre ruote non poteva mancare la bicicletta di "ladri di biciclette".

5       ne      in      ADP     EA      Gender=Masc|Number=Plur 7       case    _       _

Should be ADP/E
--------------------
# sent_id = VIT-8491
# text = Evidenziata dalle amare parole sulla trappola bosniaca in cui sono caduti i suoi duri e volontaristici propositi, ma soprattutto dalle massicce contestazioni all'estero per la ripresa degli esperimenti atomici, alle quali Chirac ha risposto ieri coll arroganza di una grandeur indisponente quanto anacronistica.

45      coll    con     ADP     EA      Number=Sing     46      case    _       _

Should be ADP/E
--------------------------
# sent_id = VIT-8944
# text = "e allora ritiro anche la mia", tuona Rosy Bindi.

7       mia     mio     PRON    P       Gender=Fem|Number=Sing|Poss=Yes|PronType=Prs    4       obj     _       SpaceAfter=No

Should be PP... maybe?  Should definitely not be just P
-----------------------
# sent_id = VIT-2999
# text = Lucca, convegno sul calzaturiero.

6       calzaturiero    calzaturiero    NOUN    s       Gender=Masc|Number=Sing 3       nmod    _       SpaceAfter=No

xpos should be S, not s
---------------------------
# sent_id = VIT-287
# text = Sono centinaia le condanne definitive pronunciate sulla base di dichiarazioni di collaboratori.

1       Sono    essere  AUX     Va      Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin   2       cop     _       _

Should be Va, not VA
```

**Comment by AngledLuffa** [comment]
Thank you for the updated dev branch.  I found two more typos and issued a pull request for them.

**Comment by AngledLuffa** [comment]
Fixed in #5
